test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -151,12 +151,10 @@ export default class BaseFormObject {
   }
 
   /**
-   * @param {string} name the text of the button to click
+   * @param {string} name the text of the save button to click
    */
   async clickSaveButton(name = 'Next') {
-    const buttonToClick = this.getButton(name);
-
-    await this.t.click(buttonToClick);
+    await this.clickButton(name);
```

Comment:
ha I wonder at what point we should eliminate the `clickSaveButton` method. doesn't have to be now, but it's clearly just a thin wrapper now.

Reply:
haha yeah that one as well as some other methods are becoming redundant and should be removed later lol